deploy_template.yaml

```diff
@@ -68,7 +68,7 @@ objects:
                     description: Floorist Query definition.
                     properties:
                       chunksize:
-                        description: 
+                        description:
```

Comment:
Can you keep this whitespace change? Automation checking for floorist image tag and regenerating this template would return it to previous state. Probably some older versions of Ruby lib (?).